egs2/voxceleb/spk1/run.sh

```diff
@@ -6,11 +6,13 @@ set -o pipefail
 
 spk_config=conf/train_RawNet3.yaml
 
-train_set="voxceleb12_devs"
+train_set="voxceleb2_dev"
 valid_set="voxceleb1_test"
 test_sets="voxceleb1_test"
+feats_type="raw"
 
 ./spk.sh \
+    --feats_type ${feats_type} \
     --spk_config ${spk_config} \
     --train_set ${train_set} \
     --valid_set ${valid_set} \
```

Comment:
I think you should add an argument `--local_data_opts "--data_dir_prefix /path/to/dir"` here since you set the default value of `data_dir_prefix` to empty in `local/data.sh`. Otherwise it will fail when running stage 1.